src/redux/locationPage/selectors.js

```diff
@@ -2,10 +2,13 @@
  * Selectors for locationPage
  */
 
+import _ from 'lodash';
```

Comment:
I think we should avoid importing all of lodash if we dont need it all. We can import submodules by doing 

```js
import { filter, includes } from 'lodash'
```

But actually since we are using a babel polyfill, we have these functions directly available on arrays and lodash is not even needed.